client/src/App.js

```diff
@@ -1,7 +1,8 @@
-import React, { useEffect } from 'react';
-import { Route, withRouter, Router } from 'react-router-dom';
+import React, { useState } from 'react';
```

Comment:
If not too much trouble, awesome if you could remove useState! Helps me debug easier. 💚

Reply:
I don't think it's possible to remove this because it is used in line 14.

Reply:
Oh okay, it popped up when I ran the app. No worries then.